docs/constants/mocha_versions.js

```diff
@@ -1,6 +1,6 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.0.0-rc10",
-  "app-latest-sha": "96b86ba8c0a11b1b854da1c256c5d737dbaeede3",
+  "app-latest-tag": "v1.0.0-rc14",
+  "app-latest-sha": "cdc6bff4cde817a0f9fd6b30b57a02ad44698d17",
   "core-latest-tag": "v1.24.0-tm-v0.34.28",
   "core-latest-sha": "d24c81afe0ecc4ccfbb7eb543a45a11794f04db4",
   "node-latest-tag": "v0.11.0-rc8",
```

Comment:
we need to change this to 
```suggestion
  "node-latest-tag": "v0.11.0-rc12",
```